app/uk/gov/hmrc/lisaapi/controllers/BulkPaymentController.scala

```diff
@@ -38,13 +38,13 @@ class BulkPaymentController @Inject()(
                                        val lisaMetrics: LisaMetrics
                                      )(implicit ec: ExecutionContext) extends LisaController {
 
-  def getBulkPayment(lisaManager: String, startDate: String, endDate: String): Action[AnyContent] =
```

Comment:
I can see the sense in this change, but I would question whether it's something we should do. It's a breaking change on an API which has existed since version 1.0. If we do decide to change this it should be versioned and we need to raise comms to let the users know we are introducing another breaking change in v2.0.

Reply:
Ok. I will raise this.